main.go

```diff
@@ -77,3 +78,11 @@ func getEnvDefault(key string, defaultVal string) string {
 
 	return val
 }
+
+func getURLUsingPlek(key string) string {
+	url, err := plek.Find(key)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return url.String()
```

Comment:
Oh that's annoying.  I wonder whether it's worth refactoring things so that urls are always instances of `url.URL`.